testing/synapse/synapse.initd

```diff
@@ -0,0 +1,15 @@
+#!/sbin/openrc-run
+
+name="synapse"
+command="/usr/bin/python3"
+command_args="-m synapse.app.homeserver --config=$config $command_args"
+
+depends() {
+	need net
+}
+
+start_pre() {
+	# /var/lib/synapse needs to be owned by the synapse user and so does the config location
+	checkpath --directory --owner synapse:synapse /var/lib/synapse
+	checkpath --directory --owner synapse:synapse $config
```

Comment:
--file

Reply:
Should synapse really own the config and not root?

Reply:
yes